specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -88,6 +88,149 @@
         ]
       },
       "x-ms-parameter-location": "method"
+    },
+    "WeatherYear": {
+      "name": "year",
+      "description": "Year of the cyclone(s)",
+      "type": "integer",
+      "format": "int32",
+      "required": true,
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "RequiredWeatherBasinID": {
+      "name": "basinID",
```

Comment:
Let's follow existing parameter naming convention to be camel case, basinId.
e.g. https://github.com/Azure/azure-rest-api-specs/blob/ef8f43c327e6e4419b0d0e41747f4eadd58e232e/specification/maps/data-plane/Weather/preview/1.0/weather.json#L485

Reply:
Updated